apps/website/src/server/utils/file-storage.ts

```diff
@@ -12,9 +12,14 @@ import { env } from "@/env";
 import { prisma } from "@/server/db/client";
 import { probeImageMeta } from "@/server/utils/probe-image-meta";
 
-function getBucketUrl() {
+const pathStyle = true; // MinIO only works with path style
```

Comment:
I mean this could easily be an env variable `FILE_STORAGE_PATH_STYLE=true` even if DO works with path style?